gooddata-java/src/main/java/com/gooddata/sdk/service/hierarchicalconfig/HierarchicalConfigValidator.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2004-2021, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.sdk.service.hierarchicalconfig;
+
+import com.gooddata.sdk.common.GoodDataException;
+import com.gooddata.sdk.model.hierarchicalconfig.SettingItem;
+
+import static com.gooddata.sdk.model.hierarchicalconfig.SourceType.*;
+
+/**
+ * Hierarchical config validator checks that setting is between allowed source types.
+ */
+public class HierarchicalConfigValidator {
```

Comment:
Can be removed - see comment in the service

Reply:
Removed